jjb/defaults.yaml

```diff
@@ -48,3 +48,6 @@
         -Djenkins
         -Dmaven.repo.local=/tmp/r
         -Dorg.ops4j.pax.url.mvn.localRepository=/tmp/r
+
+    # Nexus
+  staging-profile-id: a004df83a6249
```

Comment:
Not sure if you mean to give every project the same staging-profile-id so thought I'd ask. In OpenDaylight this is a project level setting and every project has their own id.

Reply:
for now EdgeX is using the same profile...we will revisit after Barcelona release.